src/index.js

```diff
@@ -1,19 +1,60 @@
-const addDebetInput = document.getElementById("add-debet-input");
-addDebetInput.addEventListener("click", addDebetLine);
-const addCreditInput = document.getElementById("add-credit-input");
-addCreditInput.addEventListener("click", addCreditLine);
+//this code delates inputs
+function deliteInput(fun) {
+  fun.addEventListener("click", (e) => {
+    fun.parentElement.remove();
+  });
+}
+//get debet input
+let buttonDebet = document.querySelector("#add-debet-input");
```

Comment:
Looks like buttonDebet is not modified. Use const instead of let.
Also there is typo on the line
Fix all typos in the PR and be careful with all further PRs!